src/notification/notification.component.ts

```diff
@@ -110,6 +120,17 @@ export class Notification {
 		this.close.emit();
 	}
 
+	onClick(action) {
+		if (!action) {
+			return;
+		}
+		if (isObservable(action)) {
+			(action as Subject<any>).next();
+		} else {
+			action();
+		}
```

Comment:
So the type for this is `Subject<{event: Event, action: any}> | ((event: {event: Event, action: any}) => any);`

So we should be sure to pass the DOM event and action data back 👍